django_descope/authentication.py

```diff
@@ -52,7 +53,13 @@ def authenticate(self, request: Union[HttpRequest, None], **kwargs):
             # Contains sensitive information, so only log in DEBUG mode
             logger.debug(validated_session)
         if validated_session:
-            username = validated_session[SESSION_TOKEN_NAME]["sub"]
+            try:
+                username = validated_session[SESSION_TOKEN_NAME][USERNAME_CLAIM]
+            except KeyError:
+                if settings.DEBUG:
+                    logger.debug("KeyError - USERNAME_CLAIM={USERNAME_CLAIM} does not exist on the token")
```

Comment:
should always log error (not debug)
if settings.DEBUG also throw exception so devs catch it

Reply:
something like this

```suggestion
                logger.error("Unable to authenticate user- could not find USERNAME_CLAIM={USERNAME_CLAIM} in Descope JWT") 
                if settings.DEBUG:
                    raise
```